crate/pages/sitemap.xml

```diff
@@ -1,8 +1,308 @@
-<?xml version="1.0" encoding="UTF-8"?> 
-<urlset xmlns="http://www.google.com/schemas/sitemap/0.84">{% for page in CACTUS.pages %}{% if page.path != 'error.html' %}
+<?xml version="1.0" encoding="UTF-8"?>
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9" {% for page in CACTUS.pages %}{% if page.path != 'error.html' %} xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.sitemaps.org/schemas/sitemap/0.9 http://www.sitemaps.org/schemas/sitemap/0.9/sitemap.xsd">
     <url>
         <loc>{{ page.absolute_final_url }}</loc>
         <changefreq>daily</changefreq>
         <priority>1.0</priority>
     </url>{% endif %}{% endfor %}
+<url><loc>http://crate.io/</loc><changefreq>weekly</changefreq><priority>1.00</priority></url>
```

Comment:
remove line 8-89 because they are generated automatically by Cactus

also remove all other pages that are generated by Cactus